frontend/cypress/support/index.ts

```diff
@@ -25,13 +25,14 @@ declare namespace Cypress {
       actionRequired?: string,
       justification?: string,
     ): Chainable<void>;
-    validateHWCSection(
-      section: string,
-      checkboxes: string[],
-      officer: string,
-      date: string,
-      actionRequired?: string,
-      justification?: string,
-    ): Chainable<void>;
+    validateHWCSection(params: {
```

Comment:
This is a good idea to shut down any sonar cloud complexity problems. might want to consider creating a type for this instead of using an inline definition